src/NovelRT.Interop/NrtNovelRunner.cpp

```diff
@@ -27,9 +27,11 @@ extern "C"
 
     NrtNovelRunner Nrt_NovelRunner_createCustom(int32_t displayNumber,
                                                 const char* windowTitle,
+                                                NrtWindowMode windowMode,
                                                 uint32_t targetFrameRate)
     {
-        NovelRT::NovelRunner* runner = new NovelRT::NovelRunner(displayNumber, windowTitle, targetFrameRate);
+        NovelRT::NovelRunner* runner = new NovelRT::NovelRunner(
+            displayNumber, windowTitle, static_cast<NovelRT::Windowing::WindowMode>(windowMode), targetFrameRate);
```

Comment:
`using namespace NovelRT::Windowing;` at the top of the impl file in this case? Usually I frown upon using namespace statements but we're doing it everywhere else in interop because of verbosity getting really out of hand.